documentation/src/main/asciidoc/userguide/chapters/statistics/Statistics.adoc

```diff
@@ -0,0 +1,159 @@
+`getConnectCount`:: Get the global number of connections acquired by the Hibernate Sessions (the actual number of connections used may be much smaller depending whether you use a connection pool or not).
+
+[[statistics-transaction]]
+==== Transaction statistics methods
+
+`getSuccessfulTransactionCount`:: The number of transactions that completed successfully.
+`getTransactionCount`:: The number of transactions we know to have completed.
+
+[[statistics-concurrency-control]]
+==== Concurrency Control statistics methods
+
+`getOptimisticFailureCount`:: The number of Hibernate `StaleObjectStateException`s or JPA `OptimisticLockException`s that occurred.
+
+[[statistics-entity]]
+==== Entity statistics methods
+
+`getEntityDeleteCount`:: Get the global number of entity deletes.
+`getEntityInsertCount`:: Get the global number of entity inserts.
+`getEntityLoadCount`:: Get the global number of entity loads.
+`getEntityFetchCount`:: Get the global number of entity fetches.
+`getEntityUpdateCount`:: Get the global number of entity updates.
+
+[[statistics-collection]]
+==== Collection statistics methods
+
+`getCollectionLoadCount`:: Global number of collections that were loaded.
+`getCollectionFetchCount`:: Global number of collections that were fetched.
+`getCollectionUpdateCount`:: Global number of collections that were updated.
+`getCollectionRemoveCount`:: Global number of collections that were removed.
+`getCollectionRecreateCount`:: Global number of collections that were recreated.
+
+[[statistics-query]]
+==== Query statistics methods
+
+`getQueryExecutionCount`:: Get the global number of executed queries.
+`getQueryExecutionMaxTime`:: Get the time in milliseconds of the slowest query.
+`getQueryExecutionMaxTimeQueryString`:: Get the query string for the slowest query.
+
+[[statistics-natural-id]]
+==== Natural Id statistics methods
+
+`getNaturalIdQueryExecutionCount`:: Get the global number of naturalId queries executed against the database.
+`getNaturalIdQueryExecutionMaxTime`:: Get the global maximum query time for naturalId queries executed against the database.
+`getNaturalIdQueryExecutionMaxTimeRegion`:: Get the region for the maximum naturalId query time.
+`getNaturalIdQueryExecutionMaxTimeEntity`:: Get the entity for the maximum naturalId query time.
+
+[[statistics-second-level-cache]]
+==== Second-level cache statistics methods
+
+`getSecondLevelCacheRegionNames`:: Get all second-level domain data cache region names.
+`getSecondLevelCacheHitCount`:: Global number of cacheable entities/collections successfully retrieved from the cache.
+`getSecondLevelCacheMissCount`:: Global number of cacheable entities/collections not found in the cache and loaded from the database.
+`getSecondLevelCachePutCount`:: Global number of cacheable entities/collections put in the cache.
+
+[[statistics-second-level-cache-natural-id]]
+===== Second-level cache natural id statistics methods
```

Comment:
here you use `natural id` without capitals, let's try to choose one or another.

I think all lowercase `natural id` would be better.

Reply:
Done